src/components/Footer.tsx

```diff
@@ -41,70 +47,82 @@ const Footer = () => (
     </LogoWrapper>
     <Row>
       <Col>
-        <p><strong> <StyledLink to="#"> About </StyledLink> </strong></p>
-        <p> <StyledLink to="#"> About us </StyledLink> </p>
-        <p> <StyledLink to="#"> Archives </StyledLink> </p>
-        <p> Staff </p>
-        <p> Advertise </p>
-        <p> Donate </p>
-        <p> Apparel Store </p>
-        <p> Alumni Association </p>
-        <p> Privacy Policy </p>
-        <p> Terms of Service </p>
-        <p> Contact us </p>
+        {AboutLinks.map((link, idx) => {
```

Comment:
the same `if (idx === 0) return (...) else return (...)` appears multiple times, just modularize it into a single component and pass the `links` array to the component